docs/source/demos/lead_scoring.ipynb

```diff
@@ -41,7 +41,7 @@
    "outputs": [],
    "source": [
     "lead_scoring_objective = LeadScoring(\n",
-    "    true_positives=100,\n",
+    "    true_positives=50,\n",
```

Comment:
Change needed to get the assert to pass in this file. Likely due to the new thresholding method

Reply:
I'm confused by this. If this method finds the global max, and the other found a local max, shouldn't this method give the same score or higher with the same parameters?